hwbench/environment/vendors/hpe/hpe.py

```diff
@@ -168,11 +168,23 @@ def read_power_consumption(
             )
         return power_consumption
 
-    def get_oem_chassis(self):
-        return self.get_redfish_url(
-            "/redfish/v1/Chassis/enclosurechassis/", log_failure=False
+    @cachetools.func.ttl_cache(maxsize=1, ttl=3600)
```

Comment:
Why use a TTL cache here instead of a simpler cache like the ones embedded in the python3 stdlib?

https://docs.python.org/3/library/functools.html#functools.cache

Reply:
Make sense, fixed for this one and the previous commit